CHANGES.md

```diff
@@ -17,6 +17,7 @@ Release Notes.
 * Upgrade protoc to 3.19.2.
 * Add Istio 1.13.1 to E2E test matrix for verification.
 * Upgrade Apache parent pom version to 25.
+* Use the plugin version defined by the Apache maven parent.
```

Comment:
Please be clear about all versions changed in the change logs.

Reply:
Deeply thx. I thought plugins don't need to pay attention to these, thanks again for checking

Reply:
Thanks, all dependency(dev or runtime) things are sensitive to developers. So we care.